language/move-prover/stackless-bytecode-generator/src/stackless_bytecode.rs

```diff
@@ -78,69 +107,144 @@ pub enum BranchCond {
 
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub enum Bytecode {
```

Comment:
I am puzzled by the design here.  I thought the intention was to have a label attached to every bytecode.  Instead I see that a labeled bytecode is a special variant of the general Bytecode enum.  What is the reason for this  design?

Reply:
The instructions have an AttributeId attached which is used for example for the source location. I have pulled out the branch locations from this mechanism to use traditional code labels for this use case simply because it felt cleaner.  We can change this perhaps later, but because there is already code based on this in another PR, I prefer to say with the current design, which is actually more common and vanilla for IRs.